cypress/integration/main-page.spec.js

```diff
@@ -0,0 +1,73 @@
+describe('main page',() => {
+  beforeEach(() => {
+    cy.fixture('./games.json').then((allGames) => {
```

Comment:
Love the way you did this, what exactly is allGames? is that a variable name or is it coming from somewhere? Just curious because I have dont fixtures differently.